CHANGELOG.md

```diff
@@ -65,6 +65,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * new function `working_copies()` for revsets to show the working copy commits of all workspaces.
 
+* `jj status` now supports filtering by paths. For example, `jj status .` will
+  only list changed files that are descendants of the current directory.
+
```

Comment:
maybe you'll need to move the changelog entry

Reply:
Good catch, thanks.